core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -3684,6 +3687,43 @@ private static class LogicalNotImplementor extends AbstractRexCallImplementor {
     }
   }
 
+  /**Implementor for the {@code LN, LOG, and LOG10} operators
```

Comment:
Need space at start of comment. Only the operator names should be monospaced, not “and”.